pkgs/development/tools/rust/sqlx-cli/default.nix

```diff
@@ -20,6 +20,12 @@ rustPlatform.buildRustPackage rec {
   buildInputs = lib.optionals stdenv.isLinux [ openssl ]
     ++ lib.optionals stdenv.isDarwin [ SystemConfiguration CoreFoundation Security libiconv ];
 
+  passthru.tests.version = testVersion {
```

Comment:
See  #169449

```suggestion
  passthru.tests.version = testers.testVersion {
```

Reply:
Replaced as of 61e1418f8affb9616f71d26c14e01824a722bcde